torch/library.py

```diff
@@ -91,7 +91,7 @@ class Library:
         dispatch_key: PyTorch dispatch key (default: "")
     """
 
-    def __init__(self, ns, kind, dispatch_key=""):
+    def __init__(self, ns, kind="IMPL", dispatch_key=""):
```

Comment:
It is surprising to me that the defaulting did not work when the original code does have a default

Reply:
@janeyx99 Thanks for you review! Agree! It's really better to change the docs first. Have made changes please take a look, thanks!